.github/workflows/start-updater.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+    - 'yli3/**'
+  schedule:
+  - cron: '0 */3 * * *'
+
+jobs:
+  start-updaters:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Authenticate with Google Cloud
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.9'
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.20.0'
+
+    - name: Install dependencies
+      run: pip install requests
+
+    - name: Run Python script
+      run:  python .github/workflows/scripts/updater-support-versions.py
```

Comment:
I would suggest the following flow:
- One step to download the & parse the RELEASE_VERSION file. You've already checked out the repository in L21, so you'd only need to parse it. You can pass JSON between steps to the job matrix definition (more later), so you could use `jq` (which is already installed in the Github runner). You'll set the parsed list of versions as output of this step: https://docs.github.com/en/actions/learn-github-actions/expressions#example-returning-a-json-object. 

- Then in the second job, you'll use the matrix strategy, use the checkout action for the tag (you can check out into a specific path, like `/tmp/stackrox-{version}` and run the updater there and upload the files. That way, you'd also get proper error handling (right now all your workflow runs would be green even if they failed), and you could get rid of the Python/Shell mix.